src/huggingface_hub/repocard.py

```diff
@@ -29,12 +31,21 @@
 
 # exact same regex as in the Hub server. Please keep in sync.
 TEMPLATE_MODELCARD_PATH = Path(__file__).parent / "templates" / "modelcard_template.md"
+TEMPLATE_DATASETCARD_PATH = (
+    Path(__file__).parent / "templates" / "datasetcard_template.md"
+)
+
 REGEX_YAML_BLOCK = re.compile(r"---[\n\r]+([\S\s]*?)[\n\r]+---[\n\r]")
 
 logger = get_logger(__name__)
 
 
-class ModelCard:
+class RepoCard:
```

Comment:
thoughts on making this abstract i.e. you have to actually instantiate a ModelCard or a DatasetCard? (not sure if it's even possible in Python though)

Reply:
hmm will think on this. perhaps would be nice to warn if `RepoCard` is instantiated by itself? idk

nbd either way I assume